test/instances/chainlink/CbETHBase.t.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { IERC20 } from "../../../contracts/src/interfaces/IERC20.sol";
+import { IHyperdrive } from "../../../contracts/src/interfaces/IHyperdrive.sol";
+import { IChainlinkAggregatorV3 } from "../../../contracts/src/interfaces/IChainlinkAggregatorV3.sol";
+import { ChainlinkHyperdriveInstanceTest } from "./ChainlinkHyperdriveInstanceTest.t.sol";
+
+contract CbETHBaseTest is ChainlinkHyperdriveInstanceTest {
+    /// @dev Chainlink's aggregator for the cbETH-ETH reference rate on Base.
+    IChainlinkAggregatorV3 internal constant CHAINLINK_AGGREGATOR =
+        IChainlinkAggregatorV3(0x868a501e68F3D1E89CfC0D22F6b22E8dabce5F04);
+
+    /// @dev The address of the cbETH token on Base.
+    IERC20 internal constant CBETH =
+        IERC20(0x2Ae3F1Ec7F1F5012CFEab0185bfc7aa3cf0DEc22);
+
+    /// @dev The cbETH Whale accounts.
+    address internal constant CBETH_WHALE =
+        address(0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb);
+    address[] internal vaultSharesTokenWhaleAccounts = [CBETH_WHALE];
+
+    /// @notice Instantiates the ChainlinkHyperdriveInstance testing suite with
+    ///         the configuration.
+    constructor()
+        ChainlinkHyperdriveInstanceTest(
+            InstanceTestConfig({
+                name: "Hyperdrive",
+                kind: "ChainlinkHyperdrive",
+                decimals: 18,
+                baseTokenWhaleAccounts: new address[](0),
+                vaultSharesTokenWhaleAccounts: vaultSharesTokenWhaleAccounts,
+                baseToken: IERC20(address(0)),
+                vaultSharesToken: CBETH,
+                shareTolerance: 0,
+                minimumShareReserves: 1e15,
+                minimumTransactionAmount: 1e15,
+                positionDuration: POSITION_DURATION,
+                fees: IHyperdrive.Fees({
+                    curve: 0,
+                    flat: 0,
+                    governanceLP: 0,
+                    governanceZombie: 0
+                }),
+                enableBaseDeposits: false,
```

Comment:
what's the issue with base deposits btw?

Reply:
We’re using the Chainlink integration which doesn’t support base deposits and withdrawals. The reason is that chain link oracles don’t give us a conversion mechanism from base to vault shares or vice versa. 

There may be a different way to integrate with Chainlink on base, but it seemed to me like this was the way to go (and is consistent with our wsteth integration on Gnosis Chain).

Reply:
that's right. i wrote the comment, went to eat lunch and realized it